README.md

```diff
@@ -1,10 +1,32 @@
-# somatic-snvs-indels
+# Somatic short variant analysis pipeline on Azure
+This repository is an example of running the somatic short variant analysis pipeline, based on [Best Practices Somatic Short Variant Discovery Pipeline by Broad Institute of MIT and Harvard](https://gatk.broadinstitute.org/hc/en-us/articles/360035894731-Somatic-short-variant-discovery-SNVs-Indels-), on Cromwell on Azure.<br/> 
+
+Learn more about using Azure for your Cromwell WDL workflows on our GitHub repo! - [Cromwell on Azure](https://github.com/microsoft/CromwellOnAzure).<br/>
+
+This repository is a fork from [the original](https://github.com/gatk-workflows/gatk4-somatic-snvs-indels) and has all the required changes to run the WDL workflow on Cromwell on Azure.<br/>
+
+Here, you can find the WDL files and an example inputs JSON files with links to data hosted on a public Azure Storage account. You can use the "msgenpublicdata" storage account directly as a relative path, like in the inputs JSON files. 
+
+The `mutect2.inputs.json` and `mutect2_pon.inputs.json` trigger files are ready to use. 
```

Comment:
The inputs json files are not the trigger files. Can rename "mutect2.json" to "mutect2.trigger.json" if required to be more clear

Reply:
rrenamed